number_parser/parser.py

```diff
@@ -82,24 +83,25 @@ def number_builder(token_list):
             current_grp_value = 0
             previous_token = None
```

Comment:
I think this variable is unused since we are using `previous_token = token` down below